dockerfiles/Dockerfile.jetson

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile:experimental
+#
+# This Dockerfile just installs pre-built ONNXruntime wheel inside the image.
+# Please make sure you have nvidia-runtime enabled in docker config and then build like:
+#
+# sudo -H DOCKER_BUILDKIT=1 nvidia-docker build --build-arg WHEEL_FILE=<path> -f Dockerfile.jetson
+#
+
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-base:r32.4.3
+FROM ${BASE_IMAGE} as onnxruntime
+
+ARG WHEEL_FILE
+RUN echo "Building ONNXruntime Docker image with ${WHEEL_FILE}..."
+
+# Ensure apt-get won't prompt for selecting options
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    	build-essential \
+	software-properties-common \
+	libopenblas-dev \
+        libpython3.6-dev \
+        python3-pip \
+        python3-dev \
+	cmake
+
+RUN pip3 install --upgrade pip
+RUN pip3 install setuptools wheel pybind11 pytest
+
+WORKDIR /onnxruntime
+
+# copy previously built wheel into the container
+COPY ${WHEEL_FILE} .
+
+RUN basename ${WHEEL_FILE} | xargs pip3 install 
```

Comment:
The image can be built with some modifications.
But I encountered some issues when docker run this image:

root@61db4ea75b0b:/onnxruntime# python3
Python 3.6.9 (default, Jul 17 2020, 12:50:27)
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
/usr/local/lib/python3.6/dist-packages/onnxruntime/capi/_pybind_state.py:14: UserWarning: Cannot load onnxruntime.capi. Error: 'libcublas.so.10: cannot open shared object file: No such file or directory'.
  warnings.warn("Cannot load onnxruntime.capi. Error: '{0}'.".format(str(e)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/onnxruntime/__init__.py", line 13, in <module>
    from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, \
ImportError: cannot import name 'get_all_providers'
>>>
root@61db4ea75b0b:/onnxruntime#
root@61db4ea75b0b:/onnxruntime#
root@61db4ea75b0b:/onnxruntime# cd /usr/local/lib/python3.6/dist-packages/onnxruntime/capi/
root@61db4ea75b0b:/usr/local/lib/python3.6/dist-packages/onnxruntime/capi# ldd onnxruntime_pybind11_state.so
        linux-vdso.so.1 (0x0000007fa9c13000)
        libcublas.so.10 => not found
        libcudnn.so.8 => not found
        libcurand.so.10 => not found
        libcufft.so.10 => not found
        libdl.so.2 => /lib/aarch64-linux-gnu/libdl.so.2 (0x0000007fa5ca9000)
        libpthread.so.0 => /lib/aarch64-linux-gnu/libpthread.so.0 (0x0000007fa5c7d000)
        libcudart.so.10.2 => not found
        libstdc++.so.6 => /usr/lib/aarch64-linux-gnu/libstdc++.so.6 (0x0000007fa5ae9000)
        libm.so.6 => /lib/aarch64-linux-gnu/libm.so.6 (0x0000007fa5a30000)
        libgcc_s.so.1 => /lib/aarch64-linux-gnu/libgcc_s.so.1 (0x0000007fa5a0c000)
        libc.so.6 => /lib/aarch64-linux-gnu/libc.so.6 (0x0000007fa58b2000)
        /lib/ld-linux-aarch64.so.1 (0x0000007fa9be8000)


As you can see, libcublas.so.10, libcudnn.so.8, libcurand.so.10, libcufft.so.10 and libcudart.so.10.2 are not found on the system.

root@61db4ea75b0b:/usr/lib/aarch64-linux-gnu# ls libcublas*
ls: cannot access 'libcublas*': No such file or directory
root@61db4ea75b0b:/usr/lib/aarch64-linux-gnu# cd /usr/local/cuda/
bin/     include/ lib64/   nvvm/    nvvmx/   targets/
root@61db4ea75b0b:/usr/lib/aarch64-linux-gnu# cd /usr/local/cuda/lib64
root@61db4ea75b0b:/usr/local/cuda/lib64# ls libcublas*
ls: cannot access 'libcublas*': No such file or directory

Reply:
Adding --gpus all as option for docker run can mount the missing gpu libraries and solve this problem.
You could probably mention this in README to inform users or otherwise someone like me may encounter this issue.
Thank you.

Reply:
It looks like you are not running docker with NVIDIA runtime. I will add this to instructions. Try 'nvidia-docker' instead, too. I got this: 
lab@lab-asr:~/git/onnxruntime$  sudo docker run -it --rm -v $PWD/..:/git --network host 67ccfb8cf068
root@lab-asr:/onnxruntime# cd /usr/local/lib/python3.6/dist-packages/onnxruntime/capi
root@lab-asr:/usr/local/lib/python3.6/dist-packages/onnxruntime/capi# ldd onnxruntime_pybind11_state.so
        linux-vdso.so.1 (0x0000007fb046e000)
        libcublas.so.10 => /usr/lib/aarch64-linux-gnu/libcublas.so.10 (0x0000007fa7837000)
        libcudnn.so.8 => /usr/lib/aarch64-linux-gnu/libcudnn.so.8 (0x0000007fa77f9000)
        libcurand.so.10 => /usr/local/cuda-10.2/targets/aarch64-linux/lib/libcurand.so.10 (0x0000007fa36e1000)
        libcufft.so.10 => /usr/local/cuda-10.2/targets/aarch64-linux/lib/libcufft.so.10 (0x0000007f9764e000)
        libdl.so.2 => /lib/aarch64-linux-gnu/libdl.so.2 (0x0000007f97639000)
        libpthread.so.0 => /lib/aarch64-linux-gnu/libpthread.so.0 (0x0000007f9760d000)
        libcudart.so.10.2 => /usr/local/cuda-10.2/targets/aarch64-linux/lib/libcudart.so.10.2 (0x0000007f97599000)
        libstdc++.so.6 => /usr/lib/aarch64-linux-gnu/libstdc++.so.6 (0x0000007f97405000)
        libm.so.6 => /lib/aarch64-linux-gnu/libm.so.6 (0x0000007f9734c000)
        libgcc_s.so.1 => /lib/aarch64-linux-gnu/libgcc_s.so.1 (0x0000007f97328000)
        libc.so.6 => /lib/aarch64-linux-gnu/libc.so.6 (0x0000007f971ce000)
        /lib/ld-linux-aarch64.so.1 (0x0000007fb0443000)
        libcublasLt.so.10 => /usr/lib/aarch64-linux-gnu/libcublasLt.so.10 (0x0000007f95208000)
        librt.so.1 => /lib/aarch64-linux-gnu/librt.so.1 (0x0000007f951f1000)

Reply:
Yes, 'nvidia-docker' works too.